src/humanizer.ts

```diff
@@ -49,3 +49,18 @@ export const humanizeBaseMetricValue = (
       return 'Unknown';
   }
 };
+
+/**
+ * Stringify an score into a severity string ('None' | 'Low' | 'Medium' | 'High' | 'Critical')
+ * @param score
+ */
+export const toSeverity = (score: number): Severity =>
```

Comment:
i would suggest name change to be aligned with other humanizer functions

```suggestion
export const humanizeScore = (score: number): string=>
```